src/renderer/wg_engine/tvgWgRenderData.cpp

```diff
@@ -375,21 +377,21 @@ void WgRenderDataShape::updateMeshes(WgContext &context, const RenderShape &rsha
         if (!rshape.stroke->strokeTrim(trimBegin, trimEnd)) { trimBegin = 0.0f; trimEnd = 1.0f; }
         if (rshape.stroke->trim.simultaneous) {
             for (uint32_t i = 0; i < polylines.count; i++)
```

Comment:
this is not correct. please check insta_camera.json - rounded corners are not applied

Reply:
this comment should be next to l.354...

Reply:
it is still a valid comment.
Lottie example, 1280x1280, NUM_PER_ROW 3, NUM_PER_COL 2:

current:
<img width="329" alt="Zrzut ekranu 2024-09-18 o 01 28 01" src="https://github.com/user-attachments/assets/f06021d1-36f9-4195-a263-db0ab39727bb">

expected:
<img width="267" alt="Zrzut ekranu 2024-09-18 o 01 28 10" src="https://github.com/user-attachments/assets/74e0fa9b-faf4-4938-af54-32851b0cf8d2">